src/main/resources/application-dev.properties

```diff
@@ -1,5 +1,5 @@
 spring.jpa.hibernate.ddl-auto=create
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
-spring.datasource.url=jdbc:postgresql://localhost:5432/mcitmocks_dev
+spring.datasource.url=jdbc:postgresql://localhost:5432/mcitmocks-dev
```

Comment:
We had previously changed the database name from mcitmocks-dev to mcitmocks_dev. Why is this being changed back?